README.md

```diff
@@ -4,11 +4,11 @@
 [![Downloads](https://img.shields.io/jetbrains/plugin/d/10972.svg)](https://plugins.jetbrains.com/plugin/10972)
 
 <!-- Plugin description start -->
-The Snyk Vulnerability Scanner plugin for IntelliJ helps you find and fix
+The Snyk Vulnerability Scanner plugin for IntelliJ and PyCharm helps you find and fix
 security vulnerabilities in your projects, all from within your favorite IDE.
 
 The plugin integrates seamlessly into your development environment and scans
-the open source dependencies included in your IntelliJ projects for security
+the open source dependencies included in your IntelliJand PyCharm projects for security
```

Comment:
```suggestion
the open source dependencies included in your IntelliJ and PyCharm projects for security
```

We might want to skip the "IntelliJ and PyCharm" part, because those project are "Java projects"/"Python projects" and IntelliJ is just an interface.

```suggestion
the open source dependencies included in your projects for security
```